app/views/groups/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= link_to "<<-Home", home_path, class: "btn btn-primary" %>
+<p style="color: green"><%= notice %></p>
+<%= button_to 'Sign out', destroy_user_session_path, :method => :delete, class: "btn btn-primary" %>
+
+<h1 class="text-center text-primary">Categories</h1>
+
+<div id="groups">
+  <% @groups.each do |group| %>
+    <div class="container"><%= render group %></div>
+    <h2>
+      <%= link_to "view category", operations_path, class: "btn btn-primary" %>
+    </h2>
+  <% end %>
+</div>
+<div class="addcat d-flex flex-column justify-content-around ">
+  <%= link_to "Add New Category", new_group_path, class: " btn border pb-10 mb-3" %>
+
+</div>
```

Comment:
- [ ] Sign out is not working as expected, kindly implement the feature of the logout, and therefore, the `<<home` button is directing us to the login page again which is not the logout as well.

![image](https://user-images.githubusercontent.com/55635977/154804136-94a12354-7216-4bcb-8611-3a4bef6b7423.png)